playbook/app/pb_kits/playbook/pb_dialog/_dialog.jsx

```diff
@@ -115,14 +149,28 @@ const Dialog = (props: DialogProps) => {
             overlayClassName={overlayClassNames}
             portalClassName={portalClassName}
             shouldCloseOnOverlayClick={shouldCloseOnOverlayClick}
+            status={status}
         >
-          <If condition={title}>
-            <Dialog.Header>{title}</Dialog.Header>
-          </If>
-          <If condition={text}>
-            <Dialog.Body>{text}</Dialog.Body>
-          </If>
-
+          <Dialog.Body>
+            <Flex align='center'
```

Comment:
We can't center all dialogs by default. This breaks all existing dialogs that do not want centered text.

Reply:
We also still need these conditionals that were deleted:
```
<If condition={title}>
    <Dialog.Header>{title}</Dialog.Header>
</If>
<If condition={text}>
    <Dialog.Body>{text}</Dialog.Body>
</If>
```
            
            
I'm going to try to add before/after screenshots to [the story](https://nitro.powerhrg.com/runway/backlog_items/PLAY-204) to document what is broken.